packages/utxo-lib/tests/compose.test.ts

```diff
@@ -1,27 +1,24 @@
-import { composeTx } from '../src';
+import { composeTx } from '../src/compose';
 import * as NETWORKS from '../src/networks';
 
 import { verifyTxBytes } from './compose.utils';
 import { composeTxFixture } from './__fixtures__/compose';
 import { fixturesCrossCheck } from './__fixtures__/compose.crosscheck';
 
-// keyof typeof NETWORKS;
-const getNetwork = (name?: string) =>
-    // @ts-expect-error expression of type string can't be used to index type
-    typeof name === 'string' && NETWORKS[name] ? NETWORKS[name] : NETWORKS.bitcoin;
-
 describe(composeTx.name, () => {
     composeTxFixture.forEach(f => {
-        const network = getNetwork(f.request.network);
+        const network = f.request.network ?? NETWORKS.bitcoin;
         const request = { ...f.request, network };
-        const result: any = { ...f.result };
+        const result = { ...f.result };
 
         it(f.description, () => {
-            const tx = composeTx(request as any);
+            const tx = composeTx(request);
             expect(tx).toEqual(result);
 
-            if (tx.type === 'final') {
-                verifyTxBytes(tx, f.request.txType as any, network);
+            expect(f.request.txType).not.toEqual('p2wsh');
```

Comment:
This could imho do with a short comment: why we are explicitly testing that the _**fixtures**_ don't contain a `p2wsh` type transaction? 
I can see that `verifyTxBytes` does not accept a `p2wsh` type transaction, _whatever that is_, but it's not apparent until you play around with the code.

Reply:
Oh that's fine :ok_hand:   I was just wondering if you have more insight :slightly_smiling_face: